Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js

```diff
@@ -42,16 +42,10 @@ WI.CSSKeywordCompletions.forPartialPropertyName = function(text, {caretPosition,
     if (!text.length && allowEmptyPrefix)
         return {prefix: text, completions: WI.cssManager.propertyNameCompletions.values};
 
-    let completions;
-    if (useFuzzyMatching)
-        completions = WI.cssManager.propertyNameCompletions.executeQuery(text);
-    else
-        completions = WI.cssManager.propertyNameCompletions.startsWith(text);
```

Comment:
Not removing the branch for prefix search via `CSSPropertyNameCompletions.startsWith()` because it is still used by CodeMirror editor instances in the Sources panel via `WI.CodeMirrorCompletionController`.

Ditto for `CSSCompletions.startsWith()`